rfcs/text/0000-Incident-Response-Plan.md

```diff
@@ -0,0 +1,199 @@
+### DCP PR:
+
+
+# DCP Incident Response Plan
+
+## Summary
+
+DCP needs a clear process for handling incidents that may occur in the system and path for escalating incidents, as necessary. The proposed DCP Incident Response Plan (IRP) outlines the plan and procedures for these activities.
+
+## Author(s)
+
+Sarah Tahiri [stahiri@broadinstitute.org]
+
+## Shepherd
+
+
+## Motivation
+
+Security incident response has become increasingly important as cybersecurity attacks have become more frequent, diverse, and damaging. Though preventive security measures can reduce the frequency of incidents, they cannot stop them all in their tracks. It is therefore necessary for DCP to have a documented Incident Response Plan that guides us in identifying, investigating, containing, mitigating, recovering from, and reporting an incident.
+
+Additionally, we want the DCP system to be secure and reliable for all stakeholders, including engineering teams, program governance, and, above all, end users. In order to assure that we are following security best practices, and to help us attest to that fact in the future, we are working toward compliance with the NIST SP 800-53 Rev 4 security framework. Incident response is one of the requirements of this framework.
+
+### User Stories
```

Comment:
Are reports from folks outside of the DCP team in scope? (e.g. security researchers)

Reply:
We should accept reports from all sources. The "how" part of this question is tbd.